queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/__init__.py

```diff
@@ -92,15 +92,16 @@ def create_party(business_id: int, party_info: dict, create: bool = True) -> Par
             first_name=party_info['officer'].get('firstName', '').upper(),
             last_name=party_info['officer'].get('lastName', '').upper(),
             middle_initial=party_info['officer'].get('middleInitial', '').upper(),
-            org_name=party_info.get('orgName', '').upper()
+            org_name=party_info['officer'].get('organizationName', '').upper()
         )
     if not party:
         party = Party(
             first_name=party_info['officer'].get('firstName', '').upper(),
             last_name=party_info['officer'].get('lastName', '').upper(),
             middle_initial=party_info['officer'].get('middleInitial', '').upper(),
             title=party_info.get('title', '').upper(),
```

Comment:
Title? Is this used anywhere?

Reply:
I'm not sure but it is still in the schema: https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/party.json#L98